### PR TITLE
go.mod: runtime-spec v1.1.0-rc.2

### PR DESCRIPTION
**Pull request by AkihiroSuda**
See https://github.com/opencontainers/runtime-spec/releases/tag/v1.1.0-rc.2 for the spec changes.

The `runc features` json is now defined in
https://github.com/opencontainers/runtime-spec/blob/v1.1.0-rc.2/specs-go/features/features.go

Replaces PR #3829

### CODE REVIEW

File: types/features/features.go [hunk 1]
```diff
@@ -1,122 +1,6 @@
-// Package features provides the JSON structure that is printed by `runc features` (since runc v1.1.0).
-// The types in this package are experimental and subject to change.
+// Package features provides the annotations for [github.com/opencontainers/runtime-spec/specs-go/features].
```
Comment (kolyshkin) [inline types/features/features.go:1]: This (the link) does not conform to godoc spec in https://go.dev/doc/comment#links

**Review by kolyshkin — COMMENTED** [review]
A single nit about URL formatting in godoc comment, otherwise LGTM

File: types/features/features.go (hunk 1 above)
Comment (AkihiroSuda) [reply types/features/features.go:1]: Sorry, what am I missing?

Comment (kolyshkin) [reply types/features/features.go:1]: Sorry for the noise, I haven't realized this is a doc link, i.e. a link to the Go package. Everything is correct here.

**Review by kolyshkin — APPROVED** [review]
LGTM

**Comment by AkihiroSuda** [comment]
rebased

**Comment by AkihiroSuda** [comment]
@thaJeztah @cyphar PTAL

File: docs/spec-conformance.md [hunk 2]
```diff
@@ -15,3 +15,5 @@ v1.1.0-rc.1  | `.domainname`                            | [#3600](https://github
 v1.1.0-rc.1  | `.[]mounts.uidMappings`                  | [#3717](https://github.com/opencontainers/runc/pull/3717)
 v1.1.0-rc.1  | `.[]mounts.gidMappings`                  | [#3717](https://github.com/opencontainers/runc/pull/3717)
 v1.1.0-rc.1  | `SECCOMP_FILTER_FLAG_WAIT_KILLABLE_RECV` | TODO
+v1.1.0-rc.2  | time namespaces                          | TODO
+v1.1.0-rc.2  | rsvd hugetlb cgroup                      | TODO
```
Comment (thaJeztah) [inline docs/spec-conformance.md:19]: Should we create tracking tickets for these (if there's none yet)? and link to those (after TODO)

Comment (AkihiroSuda) [reply docs/spec-conformance.md:19]: - https://github.com/opencontainers/runc/issues/3859

**Review by thaJeztah — APPROVED** [review]
LGTM, thanks!

**Pull request merged**